Source/CardStackController.m

```diff
@@ -556,15 +555,16 @@ - (void)updateCardLocationsWhileOpening {
 - (CGRect)frameForCardAtIndex:(NSUInteger)index {
     CGRect frame;
 
+    // Note: Cards at the bottom but behind the last card will be positioned 
+    // outside of the visible area, so their title bar won't show up when the 
+    // last card is being moved.
     BOOL isCardAtTheBottomAndCoveredByTheLastCard = (index > self.currentCardIndex && index < self.cards.count - 1);
```

Comment:
@attila-at-hyper like this ? https://github.com/hyperoslo/CardStack/commit/286316e11a7cedbbb4184a11cc273642a2ff9ead